src/components/content.js

```diff
@@ -698,15 +699,25 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
+    const handleData = (data) => {
```

Comment:
I'd like to refactor this code always have `handleData` be called either synchronously, or always on the next tick (`setTimeout`), instead of varying depending on the browser. I think this will lead to more predictable debugging of issues/plugins/etc.